src/pages/main/Main.jsx

```diff
@@ -287,7 +287,7 @@ const Main = () => {
                                     title={
                                         '스노우볼에 오늘의 추억이 보관되었어요!\nSNS에 링크를 공유해친구들에게 함께한 추억을 전달받아보세요☃️\n'
                                     }
-                                    url={`${import.meta.env.BASE_URL}/guest/${param.userId}`}
+                                    url={`${import.meta.env.BASE_URL}/main/${param.userId}`}
```

Comment:
이거 게스트로 가는 게 맞습니다! 다음과 같이 수정 부탁드립니다

```suggestion
                                    url={`${import.meta.env.BASE_URL}/guest/${param.userId}`}
```

사람 | 이동 위치
--|--
스노우볼 주인 | /main/userId
손님 | /guest/uerId